web/src/stream_edit.ts

```diff
@@ -257,6 +257,7 @@ export function show_settings_for(node: HTMLElement): void {
         other_settings.push(setting);
         return false;
     });
+    const realm_has_channel_folders = channel_folders.get_all_folder_ids().length > 0;
```

Comment:
How does this interact with archived folders? I expect we this to be `channel_folders.get_active_folder_ids`, so that if you have channel folders but they're all archived, this returns false.